<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>manual_beam_finder</class>
 <widget class="QDialog" name="manual_beam_finder">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>944</width>
    <height>532</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Manual beam finder</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="bottomMargin">
    <number>12</number>
   </property>
   <property name="spacing">
    <number>2</number>
   </property>
   <item row="3" column="3">
    <widget class="QSpinBox" name="integrate_position">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>300</number>
     </property>
     <property name="singleStep">
      <number>5</number>
     </property>
     <property name="value">
      <number>121</number>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QSpinBox" name="integrate_width">
     <property name="showGroupSeparator" stdset="0">
      <bool>false</bool>
     </property>
     <property name="minimum">
      <number>5</number>
     </property>
     <property name="maximum">
      <number>300</number>
     </property>
     <property name="singleStep">
      <number>5</number>
     </property>
     <property name="value">
      <number>50</number>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>integrate width</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>TOF pixels to include</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2" colspan="2">
    <widget class="QGroupBox" name="true_location">
     <property name="title">
      <string>Manual adjust</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>12</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <property name="spacing">
       <number>3</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>centre</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="true_centre">
        <property name="maximum">
         <double>221.000000000000000</double>
        </property>
        <property name="value">
         <double>121.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>fwhm</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="true_fwhm">
        <property name="minimum">
         <double>0.500000000000000</double>
        </property>
        <property name="value">
         <double>5.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="2" colspan="2">
    <widget class="QWidget" name="cross_section" native="true">
     <property name="minimumSize">
      <size>
       <width>400</width>
       <height>300</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="5" column="2" colspan="2">
    <widget class="QDialogButtonBox" name="ok">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QSpinBox" name="pixels_to_include">
     <property name="minimum">
      <number>10</number>
     </property>
     <property name="maximum">
      <number>2000</number>
     </property>
     <property name="singleStep">
      <number>10</number>
     </property>
     <property name="value">
      <number>200</number>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>integrate position</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" rowspan="6" colspan="2">
    <widget class="QWidget" name="detector_image" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>480</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>pixels_to_include</tabstop>
  <tabstop>integrate_width</tabstop>
  <tabstop>integrate_position</tabstop>
  <tabstop>true_centre</tabstop>
  <tabstop>true_fwhm</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>ok</sender>
   <signal>accepted()</signal>
   <receiver>manual_beam_finder</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ok</sender>
   <signal>rejected()</signal>
   <receiver>manual_beam_finder</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
